sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/UnconditionalJumpStatement.cs

```diff
@@ -0,0 +1,211 @@
+using System;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        public void Test1(string[] strings)

```

Comment:
I don't think we should test for that. Adding a test will mean that we duplicate the logic inside the always true/false rules, which will raise an issue anyway.